src/gpu/generic/sycl/ref_prelu.cpp

```diff
@@ -87,6 +127,8 @@ status_t ref_prelu_bwd_t::pd_t::init_conf() {
     const memory_desc_wrapper data_d(src_md(0));
     conf_.bcast_type = dnnl::impl::get_rhs_arg_broadcasting_strategy(
             *weights_d.md_, data_d);
+    conf_.reduce_diff_weights
+            = (conf_.bcast_type == broadcasting_strategy_t::scalar);
```

Comment:
Given that this is not used in the kernel I would avoid storing it in conf_.

Reply:
Done, thanks.